src/main/java/com/spotinst/sdkjava/model/SpotinstMrScalerAwsClient.java

```diff
@@ -157,6 +199,109 @@ public ApiMrScalerAws getMrScaler(ApiMrScalerAwsGetRequest mrScalerAwsGetRequest
         return retVal;
     }
 
+    /**
+     * This method is used to Get a list of all instances and instances groups in the cluster
+     *
+     *
+     * @return a list of instances
+     */
+    public List<ApiMrScalerListInstancesAws> listMrScalerInstances(
+            ApiMrScalerListInstancesRequest mrScalerListInstancesRequest) {
+        List<ApiMrScalerListInstancesAws> retVal;
+        String         clusterToGet = mrScalerListInstancesRequest.getMrScalerId();
+        RepoGenericResponse<List<ApiMrScalerListInstancesAws>> mrScalerListInstances = getSpotinstMrScalerListInstancesRepo().listMrScalerInstances(clusterToGet, authToken, account);
+
+        if(mrScalerListInstances.isRequestSucceed()){
+            retVal = mrScalerListInstances.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = mrScalerListInstances.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(String.format("Error encountered while attempting to get a list of all instances and instances groups in the cluster. Code: %s. Message: %s.",
+                                       httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+
+        return retVal;
+    }
+
+    /**
+     * This method is used to Scaler cluster
+     *
+     *
+     * @return a list of Scaler cluster
+     */
+    public List<ApiMrScalerListScalersAws> listMrScalers(ApiMrScalerListMrScalersRequest mrScalerListScalersRequest) {
+        List<ApiMrScalerListScalersAws> retVal;
+        String         clusterToGet = mrScalerListScalersRequest.getMrScalerId();
+        RepoGenericResponse<List<ApiMrScalerListScalersAws>> mrScalerListScalers = getSpotinstMrScalerListScalersRepo().listMrScalers(clusterToGet, authToken, account);
+
+        if(mrScalerListScalers.isRequestSucceed()){
+            retVal = mrScalerListScalers.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = mrScalerListScalers.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(String.format("Error encountered while attempting to get a list of Scaler cluster. Code: %s. Message: %s.",
+                                       httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+
+        return retVal;
+    }
+
+    /**
+     * This method is used to Scale Up the cluster
+     *
+     *
+     * @return a list of instances
+     */
+    public List<ApiMrScalerScaleUpAws> scaleUpMrScaler(ApiMrScalerScaleUpRequest mrScalerScaleUpRequest) {
```

Comment:
Do we need the ApiMrScalerScaleUpRequest object. We can directly pass the MrScalerId and adjustment